.circleci/config.yml

```diff
@@ -66,8 +70,15 @@ workflows:
   build_and_test:
     jobs:
       - build:
-          name: 'ruby 2.5.5'
+          name: 'ruby 2.5, ES 5.6'
```

Comment:
I shortened the versions in the name to just the point rather than the patch version to make them easier to read in CircleCi.